### PR TITLE
Third assignment: Tag Content Extractor

### PR DESCRIPTION
**Pull request by klarakufova**
Core code with the third assignment.

### CODE REVIEW

**Comment by klarakufova** [comment]
Tests added.

**Comment by klarakufova** [comment]
All incorporated.

File: src/main/java/com/mycompany/hackerrank/TagContentExtractor.java [hunk 1]
```diff
@@ -47,33 +50,41 @@ public static void main (String[] args) {
...
+            
+            if (extractor.decideIfEqualTags(frontArray, backArray)) {
+                validParts.add(part);
             }
-            validParts.add(part);
         }
         return validParts;
     }
     
+    public boolean decideIfEqualTags (String[] frontArray, String[] backArray) {
```
Comment (rsynek) [inline src/main/java/com/mycompany/hackerrank/TagContentExtractor.java]: Does the method need to be public?

File: pom.xml [hunk 2]
```diff
@@ -31,4 +31,18 @@
...
     </properties>
+    <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>2.20</version>
+        <configuration>
+          <includes>
+            <include>**/ReturnValidPartsTest.java, **/ExtractContentTest.java</include>
```
Comment (rsynek) [inline pom.xml]: This approach also requires changes when you add new test classes. Can you think of more durable solution?

File: pom.xml [hunk 3]
```diff
@@ -31,4 +31,18 @@
...
     </properties>
+    <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>2.20</version>
+        <configuration>
+          <includes>
+            <include>**/TCE*.java</include>
```
Comment (rsynek) [inline pom.xml]: I was thinking rather about organizing the tests into different packages, but this can work too. Thanks!

**Comment by rsynek** [comment]
@kkufova Please rebase/squash into a single commit and I will merge this PR. Thanks!

**Comment by klarakufova** [comment]
Done.

**Pull request merged**